src/v2/mod.rs

```diff
@@ -11,7 +11,7 @@ use concordium_contracts_common::{AccountAddress, Amount, ContractAddress, Recei
 use futures::{Stream, StreamExt};
 use tonic::IntoRequest;
 
-mod generated;
+pub mod generated;
```

Comment:
This module should be private.

Reply:
pub(crate) I assume would be ok? Same reason as below. I think you even said that I should be using the existing `types` module for these new introduced types.

Reply:
Moved types to v2 mod.